management/client.go

```diff
@@ -91,6 +91,37 @@ type ClientJWTConfiguration struct {
 	Algorithm string `json:"alg,omitempty"`
 }
 
+type Addons struct {
+	Aws                  map[string]string `json:"aws,omitempty"`
```

Comment:
Can we make `go lint` happy by capitalizing abbreviations? For example, this would become AWS, AzureSb would become AzureSB and so on...